cli/tests/test_rebase_command.rs

```diff
@@ -774,22 +763,42 @@ fn test_rebase_multiple_destinations() {
     [EOF]
     ");
 
-    let output = work_dir.run_jj(["rebase", "-r", "a", "-d", "b|c"]);
+    let output = work_dir.run_jj([
+        "rebase",
+        "--config=ui.always-allow-large-revsets=false",
+        "-r",
+        "a",
+        "-d",
+        "b|c",
+    ]);
     insta::assert_snapshot!(output, @r"
     ------- stderr -------
     Error: Revset `b|c` resolved to more than one revision
     Hint: The revset `b|c` resolved to these revisions:
       royxmykx c12952d9 c | c
       zsuskuln d18ca3e8 b | b
-    Hint: Prefix the expression with `all:` to allow any number of revisions (i.e. `all:b|c`).
```

Comment:
Would it be a problem to keep warning when `ui.always-allow-large-revsets=false`?

Here, it might make sense to also mention that the default is likely to switch to `ui.always-allow-large-revsets=true`